<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaksutapaMuokkausDlg</class>
 <widget class="QDialog" name="MaksutapaMuokkausDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maksutapa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nimi</string>
       </property>
       <property name="buddy">
        <cstring>nimiLista</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KieliListWidget" name="nimiLista">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tili</string>
       </property>
       <property name="buddy">
        <cstring>tiliCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="TiliCombo" name="tiliCombo"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="eraCheck">
       <property name="text">
        <string>Uusi tase-er√§</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Kuvake</string>
       </property>
       <property name="buddy">
        <cstring>kuvakeLista</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QListWidget" name="kuvakeLista">
       <property name="styleSheet">
        <string notr="true">QListView::item::selected {
background-color: rgb(0, 0, 255); }</string>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="gridSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TiliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/tilicombo.h</header>
  </customwidget>
  <customwidget>
   <class>KieliListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">kieli/kielilistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaksutapaMuokkausDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaksutapaMuokkausDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
